server/traces/otel_converter.go

```diff
@@ -6,37 +6,40 @@ import (
 	"math"
 	"time"
 
-	"github.com/kubeshop/tracetest/server/model"
 	"go.opentelemetry.io/otel/trace"
 	v11 "go.opentelemetry.io/proto/otlp/common/v1"
 	v1 "go.opentelemetry.io/proto/otlp/trace/v1"
 )
 
-func FromOtel(input *v1.TracesData) model.Trace {
-	return fromOtelResourceSpans(input.ResourceSpans)
+func FromOtel(input *v1.TracesData) Trace {
+	return FromOtelResourceSpans(input.ResourceSpans)
 }
 
-func fromOtelResourceSpans(resourceSpans []*v1.ResourceSpans) model.Trace {
+func FromOtelResourceSpans(resourceSpans []*v1.ResourceSpans) Trace {
 	flattenSpans := make([]*v1.Span, 0)
 	for _, resource := range resourceSpans {
 		for _, scopeSpans := range resource.ScopeSpans {
 			flattenSpans = append(flattenSpans, scopeSpans.Spans...)
 		}
 	}
 
+	return FromSpanList(flattenSpans)
+}
+
+func FromSpanList(input []*v1.Span) Trace {
```

Comment:
this was extracted as an exported function so it could be reused by the ingestor more easily